webpack.common.js

```diff
@@ -0,0 +1,33 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const path = require('path')
+const variables = require('./variables.js');
+
+module.exports = {
+    entry: "./src/app.js",
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader"
+                }
+            }
+        ]
+    },
+    plugins: [
+        new CleanWebpackPlugin(['dist']),
+        new HtmlWebpackPlugin({  // 
+            filename: 'index.html',
+            template: 'src/index.ejs',
+            title: variables.title,
+            greetings: variables.greetings
+        })
+    ],
+    output: {
+        path: path.resolve('dist'),
+        filename: 'app.js',
+        sourceMapFilename: 'sourceMap.map.js'
```

Comment:
I'm recommend you to look at [webpack.output documentation](https://webpack.js.org/configuration/output/#output-filename) (particularly using of masks like [name])